getting-started/what-you-need.md

```diff
@@ -1,102 +1,116 @@
+      * `~/Pages/_Layout.cshtml` for .NET 6
 
+### Telerik Stylesheet
 
-#### The Telerik stylesheet - For more information, refer to the [Themes]({%slug general-information/themes%}) article. Here is a short example:
+The stylesheet allows you to use one of the built-in [Themes]({%slug general-information/themes%}), for example, the Default theme:
 
 @[template](/_contentTemplates/common/js-interop-file.md#theme-static-asset-snippet)
 
 
-#### The Telerik JS Interop file - Provides features that cannot be implemented with native Blazor.
+### Telerik JS Interop File
+
+The JS Interop file provides features that cannot be implemented with native Blazor.
 
 @[template](/_contentTemplates/common/js-interop-file.md#js-interop-file-snippet)
 
 
-### Static Assets
+### Using Static Assets
 
-You can add the Telerik JS Interop file as a [static asset](https://docs.microsoft.com/en-us/aspnet/core/razor-pages/ui-class?view=aspnetcore-3.1&tabs=visual-studio#consume-content-from-a-referenced-rcl). Static assets (the `_content` folder) are automatically included in the solution from the NuGet package during build, so all you need is to reference the asset as shown in the snippet below. The `_content` folder is expanded by the framework into the local NuGet cache, and the project copies it from there.
+You can add the [Telerik JS Interop file](#telerik-js-interop-file) as a [static asset](https://docs.microsoft.com/en-us/aspnet/core/razor-pages/ui-class?view=aspnetcore-3.1&tabs=visual-studio#consume-content-from-a-referenced-rcl). Static assets (the `_content` folder) are automatically included in the solution from the NuGet package during build, so all you need is to reference the asset as shown in the snippet below. The `_content` folder is expanded by the framework into the local NuGet cache, and the project copies it from there.
 
-To enable the use of static assets in your project, add the following line to the startup file of your **Server** project:
+To enable the use of static assets in your project, add the `app.UseStaticFiles();` line to the startup file of your **Server** project:
 
  * `Startup.cs` for .NET 3.x and .NET 5
  * `Program.cs` for .NET 6
 
 **C#**
 @[template](/_contentTemplates/common/js-interop-file.md#enable-static-assets-snippet)
 
-### CDN
+### Using CDN
 
 @[template](/_contentTemplates/common/general-info.md#cdn)
 
 
->tip Telerik recommends using the [static assets](#static-assets) approach instead of a CDN. This approach relies on the static assets feature from the framework and takes the correct file from the package, so you don't have to remember to update the CDN path when upgrading to a newer version.
+>Telerik recommends using the [static assets](#static-assets) approach instead of a CDN. This approach relies on the static assets feature from the framework and takes the correct file from the package so you don't have to remember to update the CDN path when upgrading to a newer version.
```

Comment:
```suggestion
>Telerik recommends using [static assets](#static-assets) instead of a CDN. This approach relies on the static assets feature from the framework and takes the correct file from the package so you don't have to remember to update the CDN path when [upgrading to a newer version]({%slug upgrade-tutorial%}).
```